packages/lesswrong/lib/vulcan-lib/routes.ts

```diff
@@ -108,22 +117,6 @@ export const addRoute = (...routes: Route[]): void => {
   }
 };
 
-export const overrideRoute = (...routes: Route[]): void => {
```

Comment:
Did you delete this because it's not used? Perhaps also worth deleting the commented-out code that references it